test/build.js

```diff
@@ -25,11 +25,9 @@ function buildCordovaJsTestBundle (bundlePath) {
 }
 
 function collectTestBuildModules () {
-    const pkgRoot = path.join(__dirname, '..');
-
     // Add platform-specific modules that have tests to the test bundle.
     const platformModules = ['android', 'ios'].map(platform => {
-        const platformPath = path.resolve(pkgRoot, `../cordova-${platform}`);
+        const platformPath = path.dirname(require.resolve(`cordova-${platform}/package`));
```

Comment:
Would this cause COHO to not build off master/tag repo which is being released.

Reply:
No, this file is solely used to build the `cordova.js` that is used by `npm test`.

`coho` will run a grunt command that will still use whatever version of a platform lives in `..`